Polyhedron/demo/Polyhedron/Plugins/PMP/Selection_plugin.cpp

```diff
@@ -573,7 +573,8 @@ public Q_SLOTS:
       BOOST_FOREACH(Scene_polyhedron_selection_item::fg_edge_descriptor ed, selection_item->selected_edges)
       {
         selection_item->selected_facets.insert(face(halfedge(ed, poly), poly));
-        selection_item->selected_facets.insert(face(opposite(halfedge(ed, poly), poly), poly));
+        if(!is_border_edge(halfedge(ed,poly), poly))
+          selection_item->selected_facets.insert(face(opposite(halfedge(ed, poly), poly), poly));
```

Comment:
@sloriot @sgiraudot Could you review this patch? Both the title of the commit and the patch look wrong.

Reply:
it seems to work correctly, but I'll still PR a cleaner version.

Reply:
I do not know if it "works". I trust you it worked in your test.

But the description of the commit, and the patch just look wrong. In both of them the notion of "edge" and "halfedge" are completely mixed, and I wonder if the code is actually right.

Reply:
Actually I found out that commit of yours because I was searching for https://github.com/CGAL/cgal/pull/2336/commits/cba3b1c20c4f2e86f8a13ff97ba078e0d397ddd0, that was already a bug due to the confusion between edges and halfedges.

Reply:
see #2929

Reply:
... and 9708c41ad1c3b27e486f3a33c01fa4554a1899ed.

Reply:
Thanks.